pkgs/by-name/pl/plasticity/update.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq common-updater-scripts
+#shellcheck shell=bash
+
+set -eu -o pipefail
+
+version=$(curl -s ${GITHUB_TOKEN:+-u ":$GITHUB_TOKEN"} \
```

Comment:
this updates to non existant version

```
fetch https://github.com/nkallen/plasticity/releases.atom
Update 1.4.15 -> 24.0.0 in /home/kirillvr/.cache/nixpkgs-review/pr-291443/nixpkgs/pkgs/by-name/pl/plasticity/package.nix

```

Reply:
Hey, what's that `fetch` command? I've never seen it before, how should I use it? Running 
```
version=$(curl -s ${GITHUB_TOKEN:+-u ":$GITHUB_TOKEN"} \                                             jq
    curl -s https://api.github.com/repos/nkallen/plasticity/releases/latest | jq -e -r ".tag_name | .[1:]"); echo $version
```

gives me `1.4.18`. I notice that in the url, you're using `/releases` instead of `releases/latest`. [Plasticity actually has a lot of versions](https://github.com/nkallen/plasticity/releases), but the stable one is tagged latest.

Reply:
my bad, I ran nix-update which by default doesn't use `updateScript` ( lesson learned )